common/app/common/Edition.scala

```diff
@@ -98,23 +110,42 @@ object Edition {
     (__ \ "id").read[String] map (Edition.byId(_).getOrElse(defaultEdition))
   }
 
-  lazy val editionRegex = Edition.all.map(_.homePagePath.replaceFirst("/", "")).mkString("|")
+  lazy val editionRegex = allWithBetaEditions.map(_.homePagePath.replaceFirst("/", "")).mkString("|")
   private lazy val EditionalisedFront = s"""^/($editionRegex)$$""".r
 
-  private lazy val EditionalisedId = s"^/($editionRegex)(/[\\w\\d-]+)$$".r
+  private lazy val EditionalisedId = s"^/($editionRegex)/([\\w\\d-]+)$$".r
+
+  /*
+  This method is used for working out what alternate edition
+  versions of pages Google might want to know about.
+  We have to be careful that pages that use Hreflang alternates all
+  point to each other in a consistent group and that no two editions have the same locale.
 
-  def allPagesFor(request: RequestHeader): Seq[EditionLink] = {
+  https://developers.google.com/search/docs/advanced/crawling/localized-versions
+   */
+  def localizedEditionLinks(request: RequestHeader): Seq[EditionLink] = {
```

Comment:
I'm not sure I quite understand what this function is doing, do you have any examples of inputs / outputs, just so I can wrap my head around it?

Reply:
@OllysCoding 
Perhaps this bit can use further documentation.

This function generates the links to alternate pages for fronts and editionalised sections (which end up looking like this in the page source code):

```
<link rel="alternate" href="https://www.theguardian.com/uk" hreflang="en-GB" />
<link rel="alternate" href="https://www.theguardian.com/us" hreflang="en-US" />
<link rel="alternate" href="https://www.theguardian.com/au" hreflang="en-AU" />
<link rel="alternate" href="https://www.theguardian.com/international" hreflang="en" />
```

So if the input is a network front page such as `https://www.theguardian.com/uk` it would generate the links for all network front pages (including the uk), and if the input is an editionalised section such as `https://www.theguardian.com/uk/business` it would generate 

`https://www.theguardian.com/uk/business`
`https://www.theguardian.com/us/business`
`https://www.theguardian.com/au/business`

This tells Google that these pages are all localised alternatives to each other and to prioritise one over the others in search results based on what the user's locale is (this is driven by the `hreflang` attributes).

There is more detail on this in a thread on this PR which I'll @ you on.